crt_portal/cts_forms/model_variables.py

```diff
@@ -261,3 +261,9 @@
     ("AE", "Armed Forces Middle East "),
     ("AP", "Armed Forces Pacific "),
 )
+
+WHERE_ERRORS = (
+    ('location_name', 'Please enter the name of this location where this took place.'),
```

Comment:
This line should be: "Please enter the name of the location where this took place."

Also, mega-small thing, but consistent periods between the three error messages. 

I like the way you factored these into a variable by the way!

Reply:
No, there should definitely be a period at the end of each, thanks for the catch!